src/import-export/azext_import_export/generated/custom.py

```diff
@@ -0,0 +1,50 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+
+def import_export_job_list(cmd, client, resource_group_name=None):
+    if resource_group_name is not None:
+        return client.list_by_resource_group(resource_group_name=resource_group_name)
+    return client.list_by_subscription()
+
+
+def import_export_job_show(cmd, client, name, resource_group_name):
+    return client.get(job_name=name, resource_group_name=resource_group_name)
+
+
+def import_export_job_create(cmd, client, name, resource_group_name, client_tenant_id=None, location=None, tags=None,
```

Comment:
pls make sure indention correct.

Reply:
Should be no problem with indention. It's a IDE suggested style and pass our style check.